lighthouse-core/gather/gather-runner.js

```diff
@@ -467,6 +493,7 @@ class GatherRunner {
         }
         await GatherRunner.beforePass(passContext, gathererResults);
         await GatherRunner.pass(passContext, gathererResults);
+        baseArtifacts.WebAppManifest = await GatherRunner.getWebAppManifest(passContext);
```

Comment:
we have two other checks against `isFirstPass` already in this method, can we move the check for this line out here too and avoid putting it on `passContext` (and it'll be more obvious we aren't overwriting it on every turn of the loop)?

Reply:
sure, done